includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -0,0 +1,430 @@
+<?php
+
+class DesignSystemGlobalFooterModel extends WikiaModel {
+	private $data = [
+		'fandom' => [
+			'header' => [
+				'type' => 'link-image',
+				'image' => 'company/logo-fandom',
+				'href' => 'http://fandom.wikia.com',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'fandom-header'
```

Comment:
Shouldn't this be `global-footer-fandom-header`? Same thing for all message keys.